testutil/keeper/unit_test_helpers.go

```diff
@@ -226,6 +228,50 @@ func SetupForStoppingConsumerChain(t *testing.T, ctx sdk.Context,
 	require.NoError(t, err)
 }
 
+// SetupForStoppingConsumerChainChannel registers expected mock calls which assert that the channel to the consumer
+// chain is closed. To be used when we test that `StopConsumerChain` is called with `closeChan` set to `true`.
+func SetupForStoppingConsumerChainChannel(t *testing.T, ctx sdk.Context, mocks MockedKeepers) {
```

Comment:
Is there a reason why we don't just call `GetMocksForStopConsumerChain` inline to stay consistent with the rest of the repo? Ex: https://github.com/cosmos/interchain-security/blob/7986c99b3b49b7603590607e9a026a109dd5eecc/x/ccv/provider/keeper/proposal_test.go#L1077

Imo as a quick reader it's not intuitive that `SetupForStoppingConsumerChainChannel` would assert expected mocks for `StopConsumerChain`

Reply:
Renamed to `GetMocksForStopConsumerChain` to`GetMocksForStopConsumerChainWithCloseChannel` because those mocks only make sense if you call `StopConsumerChan` with `closeChan = true`.